Honeybadger/description.html

```diff
@@ -0,0 +1,56 @@
+<html>
+  
+  <head>
+    <link rel="stylesheet" type="text/css" href="preview.css">
+  </head>
+  
+  <body>
+    <!--<%def name="title()">Your Integration</%def>-->
+    <div class="sectiontitle" id="previewtitle"><h2>Tile preview</h2></div>
+    <div class="saas_tile" id="tile">
+      <div class="saas_avatar saas_in">
+        <img src="Images/200x128.png">
+      </div>
+      <div class="saas_content">
+        <p class="saas_name saas_in">Your app</p>
+        <div class="saas_controls">
+          <button class="meh">
+            <span class="js-icon text-success">+</span>
+            <span class="saas_msg" data-state_msg="Available" data-action_msg="Install">Available</span>
+          </button>
+        </div>
+        <div class="saas_description saas_in" id="tiledescription">Honeybadger tracks errors in web apps and gives you the rich context you need to diagnose and fix them faster.</div>
+      </div>
+    </div>
+    
+    <div class="integration_content" id="int-overview">
+      <h2>Overview</h2>
+      <p>Connect Honeybadger to Datadog to:</p>
+      <ul>
+        <li>See errors in the stream, in real time</li>
+        <li>Search for errors in your graphs</li>
+        <li>Discuss errors with your team</li>
+        <li>Be super awesome</li>
+      </ul>
+      <p>
+        <img class="screenshot" src="Images/snapshot.png" />
+      </p>
+    </div>
+    
+    <div class="integration_content"  id="int-configuration">
+      <h2>Configuration</h2>
+      <p>
+        <em>To capture errors from Honeybadger:</em>
+      </p>
+      <ol>
+        <li>Open your project settings in Honeybadger</li>
+        <li>Click on Alerts &amp; Integrations</li>
+        <li>Select "Datadog" as a new integration</li>
+        <li>Add your API key:
+        <pre><code><span class="hidden-api-key">api key goes here</span></code></pre>
```

Comment:
Unfortunately not because the link requires the project's ID. I'll add a link to https://app.honeybadger.io/projects, though.